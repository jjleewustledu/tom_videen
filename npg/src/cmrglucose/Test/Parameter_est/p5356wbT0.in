/data/wjp/data1/np15/p5356/p5356wb.tsc
/data/wjp/data1/np15/p5356/p5356f.dta
/data/wjp/data1/np15/p5356/p5356wbT0.pbl
/data/wjp/data1/np15/p5356/p5356wbT0a.log
/data/wjp/data1/np15/p5356/p5356wbT0b.log
/data/wjp/data1/np15/p5356/p5356wbT0c.log
/data/wjp/data1/np15/p5356/p5356wbT0d.log
