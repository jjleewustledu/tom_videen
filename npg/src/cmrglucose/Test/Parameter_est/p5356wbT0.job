./fdgsun.run <p5356wbT0.in >&p5356wbT0.out
