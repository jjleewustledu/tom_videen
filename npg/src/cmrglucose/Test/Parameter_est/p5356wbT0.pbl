0 1.87 4.11 0 3 .6 .6 .02  0 0  0.0 0.0 0.0 0.0
0 0 0 0 0 0
4
5 6 7 8  
2
1 2 
39 3.94
