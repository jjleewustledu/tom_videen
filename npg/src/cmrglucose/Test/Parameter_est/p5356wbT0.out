 
  ENTER NAME OF FILE CONTAINING BLOOD CURVE DATA
  FILE 
 /data/wjp/data1/np15/p5356/p5356f.dta                                           
   HAS BEEN OPENED, HEADER IS: 
  @01@ p5356 11-13-97 davis fdg1 11-25-97 jth                           
 
  ENTER NAME OF FILE WITH PARAMETER BLOCK INFO
  RETURN FROM 1ST PBLK CALL
  ENTER DESTINATION FOR PRIMARY PRINT-OUT
  ENTER DESTINATION FOR SECONDARY PRINT-OUT
  RETURN FROM PBLK CALL
  RETURN FROM MARQ 
     0.10000E+01    -0.99203E+00     0.97452E+00
    -0.99203E+00     0.10000E+01    -0.99430E+00
     0.97452E+00    -0.99430E+00     0.10000E+01
    -0.84741E+00     0.89704E+00    -0.92593E+00
  SINGULAR VALUES     3.8177729294048   0.17228341742221    9.4334064604879D-03
     5.1024671253917D-04
  condition no =     7482.21
  DETERMINANT OF FI MATRIX     3.1659374835551D-06
